### ISSUE TITLE
Invoice - Suggest to remove delete button from invoices page.

### ISSUE THREAD

**Issue by PeterWong1** [bitpopulous/admin_reviews#23]
Suggest to move "Delete" feature to invoice approval page (issue #10 refers).  The risk of deleting already approved invoices are mitigated.  Invoices once traded should be kept for the record whether the trade is successful or not.

![delete invoices](https://user-images.githubusercontent.com/31194353/37550915-01983f2e-29d1-11e8-9ed6-1ff6eede72aa.png)

**Comment by ArtemShatalov** [comment]
Agree. The button is removed from this page and added to Invoice details page. It's supposed to be available for invoices in two cases only: awaiting contract and awaiting auction.
https://zpl.io/aM5Wl4R
https://zpl.io/ag9Q5kQ

**Comment by topd333** [comment]
https://github.com/bitpopulous/platform-admin/issues/400
